schema.json

```diff
@@ -17,7 +17,12 @@
 			"uniqueItems": true,
 			"items": {
 				"type": "string",
-				"enum": [ "all", "at", "be", "bg", "hr", "cy", "cz", "dk", "ee", "fi", "fr", "de", "gr", "hu", "ie", "it", "lv", "lt", "lu", "mt", "nl", "pl", "pt", "ro", "sk", "si", "es", "se", "uk", "gb", "no", "is", "li", "ic", "gp", "gf", "mq", "yt", "re", "mf"]
```

Comment:
Please keep this on one line, as previously.

Reply:
Addressed this in the next commit.